docs/plugins/lifecycle.md

```diff
@@ -0,0 +1,116 @@
+# Lifecycle
+
+Via the Lifecycle plugin the current lifecycle state can be etrieved. The plugin also allows an App to change the state of the application (limited to 'close' and 'ready').
+
+The SDK also informs the App whenever the App lifecycle state changes by emitting events. You can listen to these events by using
+the `Events` plugin.
+
+## Usage
+
+Whenever you need to change or consult the current Lifecycle state, import the `Lifecycle` plugin from the SDK.
+
+```js
+import { Lifecycle } from '@lightningjs/sdk'
+```
+
+## Available states
+
+The following states are available:
+
+| State         | Description                                 |
+| ------------- |:-------------------------------------------:|
+| init          | Application is initialising                 |
+| ready         | Application is ready to be displayed        |
```

Comment:
Can we change:

Application is ready to be displayed

to

Application is ready to be displayed and pushed into the either the `active` or `background` state depending on whether it was launched by the user, or was pre-loaded by the platform.

Reply:
This came up as a confusion point today

Reply:
I think that's a little too much text for the table. Maybe we need a seperate section in the documentation that describes the lifecycle in its entirety (and not just the APIs)?